src/OpenTK/MouseCursor.cs

```diff
@@ -28,28 +28,55 @@
 #endregion
 
 using System;
+using System.Collections;
+using System.Collections.Generic;
 
 namespace OpenTK
 {
     /// <summary>
+    /// Stores information about frame timing
+    /// </summary>
+    public struct MouseCursorFrameInformation
+    {
+        /// <summary>
+        /// The frames time in seconds
+        /// </summary>
+        public float duration;
+
+        /// <summary>
+        /// The associated frame
+        /// </summary>
+        public MouseCursorFrame frame;
+    }
+
+    /// <summary>
     /// Represents a predefined or custom mouse cursor.
+    /// new MouseCursor() represents an empty animation. Use MouseCursor.Default instead.
     /// </summary>
-    public sealed class MouseCursor : WindowIcon
+    public sealed class MouseCursor : IEnumerable<MouseCursorFrameInformation>
     {
         static readonly MouseCursor default_cursor = new MouseCursor();
-        static readonly MouseCursor empty_cursor = new MouseCursor(
-            0, 0, 16, 16, new byte[16 * 16 * 4]);
+        static readonly MouseCursor empty_cursor = new MouseCursor(0, 0, 16, 16, new byte[16 * 16 * 4]);
 
-        int x;
-        int y;
+        private List<MouseCursorFrameInformation> _frames = new List<MouseCursorFrameInformation>();
 
-        MouseCursor()
+        const float DEFAULT_FRAMEDURATION = 1.0f;
```

Comment:
https://github.com/WernerWenz/opentk/blob/develop/src/OpenTK/MouseCursor.cs#L124
(Or see your own comment above)

Time is normalized to seconds.
This was the most neutral choice for me (SI base unit). 
Reason for needing to normalize is that on windows for ex. timing is done in Jiffys: https://en.wikipedia.org/wiki/Jiffy_(time)
while other OSes likely won't use this.

If I recall correct timing Information in the render loop is second based as well